web/cobrands/fixmystreet.com/fmsforcouncils.scss

```diff
@@ -492,6 +528,19 @@ $fms-green: #62b356;
     }
   }
 
+  .pricing-table__prices {
+    td {
+      background-color: #E65376;
```

Comment:
We use `#E65376` a lot in this file. I wonder whether you could make it into a variable up top, like the others? `$fms-pink` perhaps?